sig-apps/Charter.md

```diff
@@ -0,0 +1,141 @@
+Membership for roles tracked in: [sigs.yaml]
+
+- Chair
+  - Run operations and processes governing the SIG
+  - Seed members established at SIG founding
+  - Chairs *MAY* decide to step down at anytime and propose a replacement.  Use lazy consensus amongst
+    chairs with fallback on majority vote to accept proposal.  This *SHOULD* be supported by a majority of
+    SIG Members.
+  - Chairs *MAY* select additional chairs through a [super-majority] vote amongst chairs.  This
+    *SHOULD* be supported by a majority of SIG Members.
+  - Chairs *MUST* remain active in the role and are automatically removed from the position if they are
+    unresponsive for > 3 months and *MAY* be removed if not proactively working with other chairs to fulfill
+    responsibilities.
+  - Number: 2-3
+  - Defined in [sigs.yaml]
+
+- Subproject Owners
+  - Scoped to a subproject defined in [sigs.yaml]
+  - Seed members established at subproject founding
+  - *MUST* be an escalation point for technical discussions and decisions in the subproject
+  - *MUST* set milestone priorities or delegate this responsibility
+  - *MUST* remain active in the role and are automatically removed from the position if they are unresponsive
+    for > 3 months.
+  - *MAY* be removed if not proactively working with other Subproject Owners to fulfill responsibilities.
+  - *MAY* decide to step down at anytime and propose a replacement.  Use [lazy-consensus] amongst subproject owners
+    with fallback on majority vote to accept proposal.  This *SHOULD* be supported by a majority of subproject
+    contributors (those having some role in the subproject).
+  - *MAY* select additional subproject owners through a [super-majority] vote amongst subproject owners.  This
+    *SHOULD* be supported by a majority of subproject contributors (through [lazy-consensus] with fallback on voting).
+  - Number: Minimum of 3
+  - Defined in [sigs.yaml] [OWNERS] files
+
+- Members
+  - *MUST* maintain health of at least one subproject or the health of the SIG
+  - *MUST* show sustained contributions to at least one subproject or to the SIG
+  - *SHOULD* hold some documented role or responsibility in the SIG and / or at least one subproject
+    (e.g. reviewer, approver, etc)
+  - *MAY* build new functionality for subprojects
+  - *MAY* participate in decision making for the subprojects they hold roles in
+  - Includes all reviewers and approvers in [OWNERS] files for subprojects
+
+## Organizational management
+
+- SIG meets weekly on zoom with agenda in meeting notes
+  - *SHOULD* be facilitated by chairs unless delegated to specific Members
+- SIG overview and deep-dive sessions organized for Kubecon
+  - *SHOULD* be organized by chairs unless delegated to specific Members
+
+- Contributing instructions defined in the SIG CONTRIBUTING.md
+
+### Project management
+
+#### Subproject creation
+
+Subprojects may be created by [KEP] proposal and accepted by [lazy-consensus] with fallback on majority vote of
```

Comment:
So SIG Chairs would be the designated approvers of the KEP?

Reply:
I believe so. They are the ones who would make sure the lazy consensus or majority vote occurred and could carry those results to the KEP. I would consider this more of a logistical task of the chair.

Reply:
>  Is any open source tool, library, application for running apps on Kubernetes a candidate?

@pwittrock No. There has been discussion in a few places on that point. k8s, as a project, doesn't want to take on ecosystem projects. If there is something that could aide with interoperability between projects we will look at that. @brendanburns or @bgrant0607 might be able to explain this better if you need it.

When it comes to things like kubebuilder and metacontroller we've talked about where they come to that line of ecosystem enabler vs something that belongs in the ecosystem.